bin/reth/src/dirs.rs

```diff
@@ -56,6 +56,13 @@ pub fn net_dir() -> Option<PathBuf> {
     data_dir().map(|root| root.join("net"))
 }
 
+/// Returns the path to the reth secret key directory.
+///
+/// Refer to [dirs_next::data_dir] for cross-platform behavior.
+pub fn secret_key_dir() -> Option<PathBuf> {
```

Comment:
this is the secret key for p2p/network, I'd like to make this clear in the function name

Reply:
Done!